test/Libraries/TestServices/RemoteTestSessionConfig.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Configuration;
+using System.IO;
+using System.Reflection;
+
+namespace TestServices
+{
+    /// <summary>
+    /// The RemoteTestSessionConfig class is responsible for 
+    /// reading the TestServices configuration file, if provided,
+    /// to provide data for configuring a test session from a location
+    /// other than Dynamo's core directory.
+    /// </summary>
+    public class RemoteTestSessionConfig
+    {
+        public string DynamoCorePath { get; private set; }
+        public string RequestedLibraryVersion { get; private set; }
```

Comment:
For the above reasoning, `RequestedLibraryVersion` should be of type `LibraryVersion` so that it is strongly typed, the job of parsing `LibraryVersion` from `string` can be done here in the constructor (and throw an exception immediately without allowing it to propagate further downstream).